app/config/settings_prod.py

```diff
@@ -184,6 +185,9 @@
 STATIC_ROOT = BASE_DIR / 'var' / 'www' / 'stac_api' / 'static_files'
 STATICFILES_DIRS = [BASE_DIR / "spec" / "static", BASE_DIR / "app" / "stac_api" / "templates"]
 STATICFILES_STORAGE = 'whitenoise.storage.CompressedManifestStaticFilesStorage'
+# The path for the pingdom healthcheck
+HEALTHCHECK_ENDPOINT = decode_path_if_base64(os.environ.get('HEALTHCHECK_ENDPOINT', 'healthcheck'))
```

Comment:
No decoding here just take whats comes from env variable or default to 'healthcheck'